misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/CoverageReport.java

```diff
@@ -110,30 +111,29 @@ public CoverageReport(Module module, Map<String, ModuleCoverage> moduleCoverageM
     public void generateReport(JBallerinaBackend jBallerinaBackend, String includesInCoverage) throws IOException {
         String orgName = this.module.packageInstance().packageOrg().toString();
         String packageName = this.module.packageInstance().packageName().toString();
-        String version = this.module.packageInstance().packageVersion().toString();
 
         List<Path> filteredPathList;
 
         if (!module.testDocumentIds().isEmpty()) {
             filteredPathList =
                     filterPaths(jBallerinaBackend.jarResolver().getJarFilePathsRequiredForTestExecution(
-                            this.module.moduleName()), jBallerinaBackend);
+                            this.module.moduleName()), jBallerinaBackend, module.packageInstance());
```

Comment:
Shall we pass module to the filterPaths and refer to the `moduleName` and `packageInstance` in the method body?

Reply:
The moduleName is not passed to the filterPaths method. It's being used to get the required jar paths from JarResolver. So, I guess it's ok to keep this as is. WDYT?

Reply:
Ack. +1